reports/win32_os.go

```diff
@@ -13,14 +13,12 @@ func GetWin32OS() (Win32OS, error) {
 	o, err := cmd.Output()
```

Comment:
If you find that this hard to debug, you might want to add 
`cmd.Stderr = os.Stderr` right before calling `cmd.Output`. 
Doing so will capture all the original stderr output into your tool's output.

Reply:
thanks for the tip here.  I went ahead and added them to the query functions and commented the lines out so if i do have issues it will be a nice reminder.